.banzaicloud/charts/spotguide-modern-go-application/values.yaml

```diff
@@ -0,0 +1,73 @@
+# available when running the deployment in Pipeline CI
+banzaicloud:
+  tags: []
+  organization:
+    name:
+  cluster:
+    id:
+    name:
+    cloud:
+    distribution:
+    location:
+
+replicaCount: 1
+
+image:
+  repository: sagikazarmark/modern-go-application
+  tag: latest
+  pullPolicy: IfNotPresent
+
+# Minimal required configuration
```

Comment:
you should mark these with `required` where they are used and remove the defaults, I guess

Reply:
Helm install won't work that way without manually added values, will it?